### PR TITLE
IOTEVENT-285 moved deleted CassandraManaged from witness into DW-cass…

### PR DESCRIPTION
**Pull request by edwardsoo-ss**
…andra per request from https://git.vandevlab.com/iot/witness/merge_requests/11 tested locally with ticker. @charliek @Bijnagte please have a look

### CODE REVIEW

File: dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraManaged.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
+package smartthings.dw.cassandra;
+
```
Comment (charliek) [inline dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraManaged.java]: Does it make sense to write a basic test for this class?

**Comment by charliek** [comment]
Looks good to me.  Maybe we want to write a quick test since the module is already under tested?

**Comment by edwardsoo-ss** [comment]
I don't think `smartthings.dw.cassandra` has any test. I will create something

File: dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraManaged.java (hunk 1 above)
Comment (edwardsoo-ss) [inline dropwizard-cassandra/src/main/java/smartthings/dw/cassandra/CassandraManaged.java]: created some really simple tests for `smartthings.dw.cassandra`

**Comment by charliek** [comment]
Thanks! +1

**Comment by edwardsoo-ss** [comment]
@charliek thanks for the review, could you merge this for me please?

**Pull request merged**